CMakeLists.txt

```diff
@@ -22,6 +22,8 @@ set(BRION_MAINTAINER
 set(BRION_LICENSE LGPL)
 
 set(COMMON_PROJECT_DOMAIN ch.epfl.bluebrain)
+set(MVD_TOOL_DISABLE_INSTALL TRUE) # enable MVD-tool embedded usage
+
```

Comment:
It is used, the line just disable the installation of the header and the cmd line tool of MVDtool. It is a shared component, it should not be installed as part of Brion.